samples/web/content/apprtc/apprtc.py

```diff
@@ -328,77 +323,141 @@ def get_room_parameters(request, room_id, client_id, is_initiator):
 # generates non-unique numbers. We also have a special loopback client id.
 # TODO(tkchin): Generate room/client IDs in a unique way while handling
 # loopback scenario correctly.
-class Client(db.Model):
-  room_id = db.StringProperty()
-  client_id = db.StringProperty()
-  messages = db.ListProperty(db.Text)
-  is_initiator = db.BooleanProperty()
-
-# Constructs the db key for the room. We use this key to create entity groups
-# for clients in the same room, so that clients in the same room will be
-# strongly consistent.
-def get_room_key(room_id):
-  return db.Key.from_path('Room', room_id)
-
-# Creates a new Client db object and adds it to |client_map|.
-def add_client(client_map, room_id, client_id, messages, is_initiator):
-  room_key = get_room_key(room_id)
-  client = Client(room_id=room_id,
-                  client_id=client_id,
-                  messages=messages,
-                  is_initiator=is_initiator,
-                  parent=room_key)
-  client.put()
-  client_map[client_id] = client
-  logging.info('Added client ' + client_id + ' in room ' + room_id)
-
-# Removes a client from |client_map| and the datastore.
-def remove_client(client_map, client_id):
-    client = client_map.pop(client_id)
-    client.delete()
-
-# Returns clients for room.
-def get_room_clients(room_id):
-  room_key = get_room_key(room_id)
-  return Client.gql('WHERE ANCESTOR IS:ancestor AND room_id=:rid',
-                    ancestor=room_key, rid=room_id)
-
-# Returns dictionary of client_id to Client.
-def get_room_client_map(room_id):
-  clients = get_room_clients(room_id)
-  client_map = {}
-  for client in clients:
-    # Sometimes datastore converts to unicode string. This converts it back
-    # to match string coming in from request handlers.
-    client_map[str(client.client_id)] = client
-  return client_map
+class Client():
```

Comment:
nit: don't need parens. Unless you want to use new style classes and subclass object.